include/grpc/grpc_security.h

```diff
@@ -731,6 +731,14 @@ GRPCAPI int grpc_tls_credentials_options_set_cert_request_type(
     grpc_tls_credentials_options* options,
     grpc_ssl_client_certificate_request_type type);
 
+/** Set grpc_ssl_server_verification_option field in credentials options
```

Comment:
s/ssl/tls/g

Reply:
Please add a comment saying the API should only be called at a client side.

Reply:
Please also add a comment saying that if `grpc_tls_server_verification_option` is not `GRPC_SSL_SERVER_VERIFICATION`, use of a custom server authorization check (` grpc_tls_server_authorization_check_config`) will be mandatory.